src/app/meta-campaign/meta-campaign.component.ts

```diff
@@ -119,7 +119,7 @@ export class MetaCampaignComponent implements AfterViewChecked, OnDestroy, OnIni
 
   ngOnDestroy() {
     if (isPlatformBrowser(this.platformId) && this.tickerUpdateTimer) {
-      clearInterval(this.tickerUpdateTimer);
+      clearTimeout(this.tickerUpdateTimer);
```

Comment:
Looks good, I see this was set by setTimeout not setInterval so I guess clearInterval was a mistake before.